pkg/version/build_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2024 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package version
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGet(t *testing.T) {
+	commitSHA = "foobar"
+	latestVersion = "v1.2.3"
+	date = "2024-01-04"
+
+	info := Get()
+
+	assert.Equal(t, commitSHA, info.GitCommit, "Command long description should be '%s'", commitSHA)
+	assert.Equal(t, latestVersion, info.GitVersion, "Command long description should be '%s'", latestVersion)
+	assert.Equal(t, date, info.BuildDate, "Command long description should be '%s'", date)
```

Comment:
Throughout the PR the messages are redundant, the output from assert will be already informative enough
```suggestion
	assert.Equal(t, date, info.BuildDate)
```